Pod/Classes/ViewControllers/ARComponentViewController.m

```diff
@@ -26,6 +26,7 @@ - (instancetype)initWithEmission:(AREmission *)emission
 - (void)viewDidLoad;
 {
   [super viewDidLoad];
+    self.automaticallyAdjustsScrollViewInsets = NO;
```

Comment:
ideally shouldn't really be using properties in init, especially ones with side effects

did it in VS Code, will change